test/rdoc/test_rdoc_parser_ruby.rb

```diff
@@ -4297,4 +4297,39 @@ class D
     assert_equal 'A::D', a_d.full_name
   end
 
+  COERCIBLE_OPERATORS = %w[ ! != !~ % & * ** + - / < << <= <=> == === =~ > >= >> ^ | ~ ]
+
+  def test_coerce_call_seq
+    util_parser('')
+    def coercible?(op)
+      COERCIBLE_OPERATORS.include?(op)
+    end
+    def _test_coercion(op, single_arg: true, singleton: false)
```

Comment:
This test is too hard to read. There's too much logic in the tests. Please make tests "dumber" and more verbose.

Reply:
Dumbness and verbosity added.